Formula/rbspy.rb

```diff
@@ -0,0 +1,17 @@
+class Rbspy < Formula
+  desc "Sampling profiler for Ruby"
+  homepage "https://rbspy.github.io/"
+  url "https://github.com/rbspy/rbspy/archive/v0.3.2.tar.gz"
+  sha256 "8250692165937e1060ab3c94f6d4742658873073ca08ee50f6d546c7b84807ff"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
```

Comment:
Is `"--path", "."` necessary?

Reply:
It will work without it, but cargo will output the following warning:
```
warning: To build the current package use `cargo build`, to install the current package run `cargo install --path .`
```

You can see an issue related to this warning here: https://github.com/rust-lang/cargo/issues/5917

Reply:
Oh yikes, we've standardised on `system "cargo", "install", "--root", prefix` and it looks like upstream are going to kill off that syntax sooner rather than later entirely, so perhaps Homebrew needs to start changing them all 😅.

This is fine as-is then; thanks for the information!